front/app/modules/commercial/insights/admin/containers/Insights/Edit/index.tsx

```diff
@@ -90,7 +91,9 @@ const CategoryButton = styled(Button)`
     justify-content: space-between;
   }
   .buttonText {
```

Comment:
I'm wondering it this isn't too tightly coupled to component libabry's internals, doubting as classnames could easily be considered a public interface, but let's make sure that's something we can call a public interface of the library.

Reply:
That's a very good point! Unforunately, I don't see an easy way to style the button without referencing `buttonText` atm, because it is the node that needs to be 100% for the children to be positioned correctly. And my impression is that this is the reason the buttonText class name was added in the first place. Seems like we need to maybe rethink the button component in the component library on multiple fronts.